### ISSUE TITLE
Log skipped and up-to-date projects in counts

### ISSUE THREAD

**Issue by mhutch** [mono/monodevelop#5431]
If I build a solution in VS where one project is up to date and one is disabled, I get the following message:

```
1>------ Skipped Build: Project: NetStandardLib, Configuration: Debug Any CPU ------
1>Project not selected to build for this solution configuration 
========== Build: 0 succeeded, 0 failed, 1 up-to-date, 1 skipped ==========
```

MD only logs the succeed/fail count, it does not log the number of skipped or up to date projects.

> VS bug [#650153](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/650153)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.8.0.184 (master)

Author: Mikayla Hutchinson
Commit: 429ecb52e84709526e173d72b25d663f98db562f (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.7.0.1070 (release-7.7)

Author: Mikayla Hutchinson
Commit: 429ecb52e84709526e173d72b25d663f98db562f (mono/monodevelop)
